### PR TITLE
DOC-2368: Update bundling documentation for all premium plugins for new index.js files.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2368

Site: [Staging branch](http://docs-feature-71-doc-2368.staging.tiny.cloud/docs/tinymce/latest/bundling-plugins/)

Changes:
- [x] Stage 1: `Update` bundling-plugins.adoc to reflect simplified bundling.
- [x] Stage 2: `Update` integrations to fall inline with these changes.

Pre-checks:
- [x] Branch prefixed with `feature/7/` or `hotfix/7/`

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/partials/plugin-files/plugin-file-list-emoticons.adoc [hunk 1]
```diff
@@ -1,7 +1,6 @@
-.Base js file
+.Requires other js files
 [source, js]
 ----
-./plugins/emoticons/plugin.js
 ./plugins/emoticons/js/emojiimages.js
 ./plugins/emoticons/js/emojis.js
```
Comment (TheSpyder) [inline modules/ROOT/partials/plugin-files/plugin-file-list-emoticons.adoc:5]: This was misleading before - the integrator does not require both files, only the file matching their `emoticons_database` setting
https://www.tiny.cloud/docs/tinymce/latest/emoticons/#emoticons_database

Comment (kemister85) [reply modules/ROOT/partials/plugin-files/plugin-file-list-emoticons.adoc:5]: @TheSpyder Can you confirm if we `remove` or `leave` these? I thought that these files had to be included based on our discussion last week? Maybe I misunderstood 🤣 .

**Review by spocke — APPROVED** [review]

File: modules/ROOT/partials/plugin-files/plugin-file-list-emoticons.adoc (hunk 1 above)
Comment (TheSpyder) [reply modules/ROOT/partials/plugin-files/plugin-file-list-emoticons.adoc:5]: Adjust them. Note that one or the other is required, depending on which one is chosen through the `emoticons_database` setting.

**Review by danoaky-tiny — APPROVED** [review]

File: modules/ROOT/pages/bundling-plugins.adoc [hunk 2]
```diff
@@ -7,17 +7,11 @@
...
-
-Required files for an example plugin:
+include::partial$plugin-files/plugin-file-index.js.adoc[]
 
-[source, js]
-----
-./plugins/example/content.js // or './plugins/example/content_css.js'
-./plugins/example/plugin.js
-----
+The following table shows examples of the syntax used to bundle the following plugin.
```
Comment (ltrouton) [inline modules/ROOT/pages/bundling-plugins.adoc:12]:
```suggestion
The following table shows examples of the syntax used to bundle the "<plugincode>" plugin.
```

File: modules/ROOT/pages/bundling-plugins.adoc [hunk 3]
```diff
@@ -7,17 +7,11 @@
...
 
-Example syntax for including the example "plugin" in a bundle using `content.js` or `content_css.js` files for bundling:
+Example syntax for including the example "plugin" in a bundle using `content.js`for bundling:
```
Comment (ltrouton) [inline modules/ROOT/pages/bundling-plugins.adoc:14]:
```suggestion
Example syntax for including the example "<plugincode>" plugin in a bundle using `content.js`for bundling:
```

File: modules/ROOT/pages/bundling-plugins.adoc (hunk 2 above)
Comment (ltrouton) [reply modules/ROOT/pages/bundling-plugins.adoc:12]: Now that I think about it, do we need this paragraph if we have one below noting the table?

Comment (kemister85) [reply modules/ROOT/pages/bundling-plugins.adoc:12]: Good point, lets remove this paragraph

Comment (kemister85) [reply modules/ROOT/pages/bundling-plugins.adoc:12]:
```suggestion
```

File: modules/ROOT/partials/plugin-files/plugin-file-list-emoticons.adoc (hunk 1 above)
Comment (kemister85) [reply modules/ROOT/partials/plugin-files/plugin-file-list-emoticons.adoc:5]: resolved in last commit to branch.

File: modules/ROOT/pages/bundling-plugins.adoc [hunk 4]
```diff
@@ -7,17 +7,13 @@
 include::partial$module-loading/bundling-ref-example.adoc[]
 :!editorcomponent:
 
-The following table shows examples of the syntax used to bundle the following plugin.
+[NOTE]
+If using {productname} 7.0 or earlier, please refer to the link:https://www.tiny.cloud/docs/tinymce/6/bundling-plugins/[Bundling TinyMCE plugins using module loading] from the {productname} v6 documentation guide.
```
Comment (ltrouton) [inline modules/ROOT/pages/bundling-plugins.adoc:11]:
```suggestion
If using {productname} 7.0.1 or earlier, please refer to link:https://www.tiny.cloud/docs/tinymce/6/bundling-plugins/[Bundling TinyMCE plugins using module loading] from the {productname} v6 documentation guide.
```

**Review by ltrouton — APPROVED** [review]

**Review by ShiridiGandham — APPROVED** [review]
LGTM

**Review by MitchC1999 — APPROVED** [review]

**Pull request merged**